### DISCUSSION TITLE
Review Thread: Mailbox.org

### DISCUSSION THREAD

**Discussion by lissy93** [Lissy93/email-comparison#29] [Reviews]
<h1 align="center"><sup>User Reviews</sup><br>Mailbox.org</h1>

This is a place for you to share your thoughts and experiences using this email provider. The aim is to help other users make an informed discussion when choosing a private and/or secure email service.

Don't forget to upvote the reviews you've found helpful!

<p align="center">
<sup>Click the card below, to view info about this email service</sup>
<a href="https://email-comparison.as93.net/email/mailbox.org">
<img width="800" src="https://api.apiflash.com/v1/urltoimage?access_key=97d02977152a442a82c708d3efe53e41&url=https%3A%2F%2Femail-comparison.as93.net%2Femail%2Fmailbox.org%2Fsummary&format=png&width=1500&ttl=2592000&response_type=image&element=body" />
</a>
</p>

---

<blockquote>
<h3>Important</h3>
<sup>
    If you are associated with a given service or it's direct competitors in an form, you MUST disclose your affiliation<br>
    Please note that your review will be visible on the website, along with your GitHub name, profile age and avatar<br>
    No review will ever be hidden / deleted, unless it is obviously spam or actively offensive<br>
    By posting your review, you agree to follow the Contributor Code of Conduct<br>
    Basic markdown formatting is supported (links, scaled images, bold, italics)<br>
</sup>
</blockquote>

**Comment by admonstrator** [comment]
I have been using mailbox.org for at least 6 years now, and I am still totally happy. This is, in my opinion, one of the best mail providers within Germany. Like always, you need to understand that privacy is something you will pay for - that's why this service isn't free. 

With a paid account, you can use your own domain or even create wildcard (catch-all) email addresses. If you don't like this, you can still go with an alias (which enables you to send from this alias as well). For a few months, not only PGP but S/MIME is integrated into the web mailer as well; even with bring-your-own-key. For people who likes to register to different services without providing "valid" addresses, you can go with temporary created addresses which you can delete if you don't need them anymore.

All in all, this one is my favorite provider and the support is fast, friendly and helpful as well. Oh, and everything works fine on iOS as well; even with push.
